examples/kubeflow/mnist/Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: docker-build docker-push mnist
+
+docker-build:
+	docker build -t ysimonson/mnist_pachyderm_pipeline:v1.1.9 .
```

Comment:
I am assuming "ysimonson" is only for testing right now.

Reply:
Yes, will swap that over once changes to the image are done